Extensions/TileMap/tilemapruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
I see this file was autoformatted 👍 But it seems it was run with a .prettierrc configuration that was not the one that is in `Extensions/.prettierrc` - as I can see there is no semicolon at all in this file. Can you change your configuration and run again autoformatting?

Thanks! :)